game/roomserver.go

```diff
@@ -71,9 +71,10 @@ func dbExists(path string) bool {
 // InitTables initializes tables
 func InitTables() {
 	query := "[" +
-		"\"CREATE TABLE `round_room_mapping` (`round_id` INTEGER NOT NULL, `room_id` INTEGER NOT NULL, UNIQUE (`room_id`) ON CONFLICT REPLACE);\"," +
+		"\"CREATE TABLE `round_room_mapping` (`round_id` INTEGER NOT NULL, `room_id` INTEGER NOT NULL, UNIQUE (`room_id`, `round_id`) ON CONFLICT REPLACE);\"," +
```

Comment:
So this is not needed.. there is only one room_id... because this DB is room specific.. so if there is an update with round id .. it will replace the only entry.
I changed this from UNIQUE (`room_id`, `round_id`) to UNIQUE (`room_id`) because for some reason the db was adding two entries in the table.